### PR TITLE
Repo structure

### PR DESCRIPTION
**Pull request by prestwick**
- Readme for repo and each example
- Directory and LV project for each example
- Gitignore all the cache files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*.gcompcache
+*.lvprojectcache
+*.niaddoncache
+*.lvcodedb
+*.aliases
+*.lvlps
+*.gcompcache-journal
```
Comment (rajsite) [inline .gitignore]: Might want to add `Builds` and `*.lvcompile`

Would be great to get these to build on Travis and publish to gh-pages some day, but not today :)

Comment (rajsite) [inline .gitignore]: Looks like git is already ignoring directories starting with `.` like `.cache` and the generated `PROJECTMEDIA` dir is empty and ignored by git

File: Readme.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # webvi-examples
+- Please use all of the default namespaces and directory names when building the Web Application in each example
+- Fork or branch this repo and work on a single example in a single branch at a time
+- Draft the readme for each example as you go. The Readme doesn't need to be perfect, but there must be something before we can pull it in.
+
```
Comment (rajsite) [inline Readme.md]: Can you point to an example README for the example READMEs?

ie what sections should they have?

setup
dependencies, ie NXG, LV2016, etc
overview
description
usage
expected behavior

Comment (rajsite) [inline Readme.md]: An example is good so they have consistent markdown formatting too

File: .gitignore (hunk 1 above)
Comment (prestwick) [inline .gitignore]: I'm leaving `Builds` in due to the fact we don't know how to do LV builds on CI infrastructure (maybe a good hackathon topic ;-). 

Git wasn't ignoring the .cache directory. In fact I commit all those files while troubleshooting another issue with Jaidev. See https://github.com/prestwick/niweek-hero-2017/tree/mod-gitignore

Comment (rajsite) [inline .gitignore]: Ah I see. Did you enable Github Pages for the master branch then?

If you do that would be good to add an index.html at the repo root that link to the different examples.

I don't think we have any files starting with an underscore but just to be safe you may also want to add an empty .nojekyll file to the repo root so Github Pages doesn't try to get all magic-y on you (if you enable Github Pages on master).

**Comment by prestwick** [comment]
I added a readme to one of the examples that I had mostly done any way. It still needs some work, but I think its good enough as an example.

**Pull request merged**